CHANGELOG.md

```diff
@@ -4,10 +4,19 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [Unreleased]
+## 0.1.0
 
 ### Added
 
-- Automatically create tasks based on the targets in a package.
+- Automatically create build tasks based on the targets in a package.
 - Show package dependencies in the Explorer.
+- Add button to package dependencies view to update packages.
 - Resolve dependencies when **Package.swift** or **Package.resolved** changes.
+- Integrated with Soucekit-LSP.
+- Generate launch configurations for each executable and tests in Package.swift.
+- Added "Swift" output channel providing a history of all actions.
```

Comment:
```suggestion
- Add "Swift" output channel providing a history of all actions.
```

Reply:
Thinking about it. Shouldn't all the lines be in the past tense. Given the section is called "Added".